src/stirling/testing/demo_apps/go_https/server/https_server.go

```diff
@@ -19,20 +19,24 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"io"
 	"log"
 	"net/http"
 
-	"github.com/spf13/pflag"
-	"github.com/spf13/viper"
+	"golang.org/x/net/http2"
 )
 
 const (
 	httpPort  = 50100
 	httpsPort = 50101
 )
 
+// Import the http2 package to ensure golang.org/x/net exists within the binary's
+// buildinfo.
+var s http2.Server //nolint:unused
+
```

Comment:
you can just change the import to `_ "golang.org/x/net/http2"` instead

Reply:
Though this feels a bit odd to me and indicates that maybe there's a gap in how we read or utilize buildinfo?